### PR TITLE
Adds Javadoc for Places related class

### PR DESCRIPTION
**Pull request by felixhalim**
Places related class includes:
- `AddPlaceCommand.java`
- `DeletePlaceCommand.java`
- `ExitCommand.java`
- `Place.java`
- `PlaceList.java`

### CODE REVIEW

File: src/main/java/seedu/trippie/command/AddPlaceCommand.java [hunk 1]
```diff
@@ -26,6 +26,12 @@
     private final int start;
     private final int end;
 
+    /**
+     * Adds the place to the lsit of places to be visited.
```
Comment (weishuangtan) [inline src/main/java/seedu/trippie/command/AddPlaceCommand.java:30]: Typo for "list".

**Review by weishuangtan — APPROVED** [review]
LGTM!

File: src/main/java/seedu/trippie/command/AddPlaceCommand.java (hunk 1 above)
Comment (felixhalim) [reply src/main/java/seedu/trippie/command/AddPlaceCommand.java:30]: Thank you for pointing that out 👍

**Review by ivanderjmw — APPROVED** [review]
LGTM

**Pull request merged**